src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,52 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY_CODENAME = "supply";
+    private static final String SEPARATOR = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] readData = readFromFile(fromFileName);
+        int supply = 0;
+        int buy = 0;
+        for (int i = 0; i < readData.length; i += 2) {
+            if (readData[i].equals(SUPPLY_CODENAME)) {
+                supply += Integer.parseInt(readData[i + 1]);
+            } else {
+                buy += Integer.parseInt(readData[i + 1]);
+            }
+        }
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append("supply,").append(supply).append(System.lineSeparator())
+                .append("buy,").append(buy).append(System.lineSeparator())
+                .append("result,").append(supply - buy).append(System.lineSeparator());
```

Comment:
counting stock and report creation are different actions. lets create a separate method for report creation